README.md

```diff
@@ -21,36 +21,60 @@ plugins {
     * apply flyway migrations from `src/main/resources/db/migration`
     * run jOOQ code generator with a configuration tailored for Kotlin Projects
 * Make the `compileKotlin` task depend on `generateJooq`
-* Add the generated jooq code to the main source set
+* Add the generated jOOQ code to the main source set
 
 > [!NOTE]
 >
 > This plugin requires a `docker` installation
 
 ## Configuration
 
-In general, this gradle plugin aims to require as little configuration as possible.
+In general, this gradle plugin aims to require as little configuration as possible and all configuration is optional.
 
-However, one may configure the jOOQ generation as follows:
+But here are the available configuration options with their default values:
 
 *build.gradle.kts*
-
 ```kotlin
 import com.optravis.jooq.gradle.ContainerConfig
 import com.optravis.jooq.gradle.ExperimentalJooqGeneratorConfig
+import com.optravis.jooq.gradle.GeneratorType
+import com.optravis.jooq.gradle.JooqDatabaseConfig
 import com.optravis.jooq.gradle.JooqGeneratorConfig
 
+
+// If defined, the `group` determines the package name of the generated code 
+group = "org.jooq.generated"
```

Comment:
I'm nor sure if this is a good example. Generally speaking, the `group` should have something to do with the project. Maybe a better example would be
```
group = "my.project"
```
?

Reply:
I agree

Reply:
Yes, that's true. Here my intent was not to suggest what one should write, but rather what is the default if not set.

But after a discussion with @ursjoss, we decided to remove the default value of `org.jooq.generated` as that's misleading. Instead, the task will fail if neither the group, nor the package name is defined.

And I will apply your suggestion for the readme @bartkummel ;-)